gremlin-javascript/src/main/javascript/gremlin-javascript/.eslintrc.js

```diff
@@ -22,18 +22,17 @@
 module.exports = {
   env: {
     commonjs: true,
-    es6: true,
+    es2022: true,
     node: true,
   },
   parserOptions: {
-    ecmaVersion: 2021,
+    ecmaVersion: 2022,
   },
   ignorePatterns: ['test/**/*.js', 'doc/**/*.js'],
   extends: ['eslint:recommended', 'prettier'],
   plugins: ['prettier'],
   rules: {
     'prettier/prettier': ['error', { endOfLine: 'auto' }],
-    indent: ['error', 2, { SwitchCase: 1 }],
```

Comment:
@tien Are the changes to the ESLint rules necessary? I'm specifically concerned about removing the `indent` rule.

Reply:
Yes, there was an error in this part. Where ESLint was conflicting with prettier.
```ts
    this._ws = new WebSocket(
      this.url,
      globalThis.WebSocket === undefined
        ? {
            headers: headers,
            ca: this.options.ca,
            cert: this.options.cert,
            pfx: this.options.pfx,
            rejectUnauthorized: this.options.rejectUnauthorized,
            agent: this.options.agent,
          }
        : undefined,
    );
```

Reply:
Ah, Prettier is handling things. Nice catch.